src/main/kotlin/model/OSPlacesAPI.kt

```diff
@@ -0,0 +1,25 @@
+package uk.gov.justice.laa.architecture
+
+import com.structurizr.model.Model
+import com.structurizr.model.SoftwareSystem
+import com.structurizr.view.ViewSet
+
+class OSPlacesAPI private constructor() {
+  companion object : LAASoftwareSystem {
+    lateinit var system: SoftwareSystem
+
+    override fun defineModelEntities(model: Model) {
+      system = model.addSoftwareSystem("OS Places", "Postcode lookup API").apply {
```

Comment:
```suggestion
      system = model.addSoftwareSystem("OS Places API", "Ordnance Survey postcode lookup API").apply {
```
Just in case people don't know what OS means. :)

Reply:
Good point. Fixed 😁